examples/kube-state-metrics.yaml

```diff
@@ -0,0 +1,287 @@
+# Copyright 2021 Google LLC
```

Comment:
nit: we're in 2022

Reply:
Yeah - let's fix the header in a separate PR